validator/js/nodejs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amphtml-validator",
-  "version": "1.0.33",
+  "version": "1.0.34",
```

Comment:
Is this version change necessary to get the PR checks to pass? If not, then best to separate it into a different PR.

Reply:
@twifkak Not at all, it just got updated. I'll revert it.